support-models/src/main/scala/com/gu/support/workers/states/GetPaymentMethodState.scala

```diff
@@ -0,0 +1,19 @@
+package com.gu.support.workers.states
+
+import java.util.UUID
+import com.gu.support.encoding.Codec.deriveCodec
+import com.gu.support.workers.{User, _}
+
+case class GetPaymentMethodState(
```

Comment:
Does it make sense to define a new type here? 

If I understand it correctly, the JSON from support-frontend could be passed to this lambda _or_ CreatePaymentMethod, which means that anyone who wants to add new input state needs to remember to update (and test) both scenarios.

It might be less error prone to define a single type `StartExecutionState` (or similar), which is passed to both lambdas?